calico-cloud/operations/cluster-management.mdx

```diff
@@ -8,9 +8,33 @@ description: Manage cluster storage, add new clusters, and configure alerts.
 
 To connect new clusters to {{prodname}}, select Managed Clusters in the left nav, and click **Connect Cluster**.
 
-## Recover a deleted managed cluster
+## Delete a managed cluster
 
-If you inadvertenty delete a managed cluster, rerun the `curl` command and the cluster will reconnect to {{prodname}}.
+### Prerequisites
+
+- Only users with either the `Owner` or `Admin` [roles](../users/user-management.mdx) are permitted to delete managed clusters.
+- The managed cluster must not be connected. For help with disconnecting your managed cluster, see [how to uninstall {{prodname}}](disconnect.mdx).
```

Comment:
* The cluster you want to remove is [disconnected from Calico Cloud](disconnect.mdx).

Reply:
Item updated https://github.com/tigera/docs/pull/961/commits/3ae147cdd1e8883739cc8aec4233aa4e82340c0f